### PR TITLE
Removed snippet state published (UI)

### LINKED ISSUE #1099: Snippet state is not considered while publishing pages
Also add `state` as list column.

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [x] test coverage
- [ ] gather feedback for my changes
- [x] Default value of state in controller to published
- [x] Remove dropdown and state in request
- [x] Upgrade command which set all state to published

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1099
| BC breaks        | snippet state to published => see CHANGELOG.md
| Documentation PR | none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,8 @@ CHANGELOG for Sulu
 ==================
 
 * 1.0.0-RC2 (2015-06-17)
-    * BUGFIX      #1264 [ContentBundle]  fixed save of changed block type
```
Comment (wachterjohannes) [inline CHANGELOG.md]: fixed typo

File: UPGRADE.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
 
+## dev-develop
+
+### Snippets
+
+Snippet state has been removed and set default to published. Therefor all snippets has to be set to published by this
+running this command:
+
+```bash
+app/console sulu:upgrade:rc3:snippet-state
```
Comment (danrot) [inline UPGRADE.md]: Interferes with https://github.com/sulu-io/sulu/issues/819 ... But I guess we will also just remove this command before 1.0

**Comment by chirimoya** [comment]
@wachterjohannes @danrot ... can be merged?

**Comment by wachterjohannes** [comment]
yeah i think so (= i will rebase it!

**Comment by danrot** [comment]
As this PR didn't make it into RC3, and we should delete all upgrade commands before 1.0 (https://github.com/sulu-io/sulu/issues/819) I'd suggest to provide a SQL2 snippet for this task. (The command isn't working atm anyway :wink:)

**Comment by wachterjohannes** [comment]
@danrot added

**Pull request merged**